bob.js

```diff
@@ -1142,6 +1153,20 @@ isILRace = () => {
     return categoryName === "Individual Levels";
 }
 
+//e.g. 1 --> "  1"
+addSpaces = (input, outputLength) => {
+    var spaceCount = outputLength - input.length;
+    if (spaceCount > 0) {
+        var spacesString = " ";
+        var i = 1;
+        for (; i < spaceCount; i++) {
```

Comment:
Can we combine these two lines so it's `for (var i = 1; i < spaceCount; i++) {` ?

Reply:
I don't know, this is how w3schools wrote it